exporters/trace/src/test/java/com/google/cloud/opentelemetry/trace/TraceTranslatorTest.java

```diff
@@ -0,0 +1,198 @@
+package com.google.cloud.opentelemetry.trace;
+
+import com.google.devtools.cloudtrace.v2.AttributeValue;
+import com.google.devtools.cloudtrace.v2.Span;
+import com.google.devtools.cloudtrace.v2.TruncatableString;
+import com.google.rpc.Status;
+import io.opentelemetry.common.Attributes;
+import io.opentelemetry.common.ReadableAttributes;
+import io.opentelemetry.sdk.trace.data.SpanData;
+import io.opentelemetry.trace.Span.Kind;
+import io.opentelemetry.trace.*;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.util.*;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.*;
+
+@RunWith(JUnit4.class)
+public class TraceTranslatorTest {
+
+  @Test
+  public void toDisplayNameTest(){
+    String serverPrefixSpanName = "Recv. mySpanName";
+    String clientPrefixSpanName = "Sent. mySpanName";
+    String regularSpanName = "regularSpanName";
+    Kind serverSpanKind = Kind.SERVER;
+    Kind clientSpanKind = Kind.CLIENT;
+
+    assertEquals("Recv. mySpanName", TraceTranslator.toDisplayName(serverPrefixSpanName, serverSpanKind));
+    assertEquals("Sent. mySpanName", TraceTranslator.toDisplayName(clientPrefixSpanName, clientSpanKind));
+    assertEquals("Recv.regularSpanName", TraceTranslator.toDisplayName(regularSpanName, serverSpanKind));
```

Comment:
It would just return the string you sent in (namely, "Recv.regularSpanName"). The server/clientSpanKind are used to check that the string does NOT already start with "Recv." or "Sent." respectively. 
For example, if the method is passed in with parameter serverSpanKind, the method will add the literal "Recv." in front of the passed in string unless the string already starts with "Recv."